pepper-apis/src/main/java/org/broadinstitute/ddp/json/dsm/DsmNotificationPayload.java

```diff
@@ -19,6 +20,9 @@
     @SerializedName("eventDate")
     private long eventDate;
 
+    @SerializedName("eventData")
+    private JsonElement eventData;
```

Comment:
Using `JsonElement` for flexibility since other event types might have a different structure for their data.